lib/trento/support/commanded_utils.ex

```diff
@@ -0,0 +1,36 @@
+defmodule Trento.Support.CommandedUtils do
+  @moduledoc false
+  alias Trento.ActivityLog
+
+  def commanded, do: Application.fetch_env!(:trento, Trento.Commanded)[:adapter]
+
+  def maybe_correlated_dispatch(command, ctx \\ :uncorrelated_dispatch) do
```

Comment:
What about renaming this to `correlated_dispatch/2` as well?
The `maybe_` thing looks a bit odd in public functions.

In fact, do we have any use case where we don't set the 2nd parameter?
I don't see any, in that case, they user could use `dispatch` itself, right?